contracts/QuestFactory.sol

```diff
@@ -556,6 +565,13 @@ contract QuestFactory is Initializable, OwnableUpgradeable, AccessControlUpgrade
         getMintFeeRecipient().safeTransferETH(mintFee);
     }
 
+    function setNFTQuestFeeList(address[] calldata toAddAddresses, uint[] calldata fees) external onlyOwner
+    {
+        for (uint i = 0; i < toAddAddresses.length; i++) {
+            NFTQuestFeeList[toAddAddresses[i]] = NFTQuestFees(fees[i], true);
```

Comment:
Should we emit event for this? My reasoning is that it would be easier to track onchain for fee set activity